### DISCUSSION TITLE
🚢 Shipped: Export Project View to CSV ⛴️ [Issues & Projects Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#60798] [Projects and Issues]
The power of your feedback continues, thanks to everyone who requested the export project view to CSV feature ([here](https://github.com/orgs/community/discussions/47733) and [here](https://github.com/orgs/community/discussions/6584)) - it is now available and we want to hear what you think! We’re not just shouting out those folks, but even more below. 

🦸‍♂️ **Incredible in Issues and Projects**: A round of applause for these community citizens 👏
* @spenserblack: For their [workaround](https://github.com/orgs/community/discussions/60170#discussioncomment-6367697) for Issue Template labels (bonus points for it being an Actions workflow) and additional suggestions. 
* @akash-singh8: Coming up with the right search syntax isn’t always easy, luckily they came to the rescue with syntax to find [Issues that don’t have the phrase "RELEASE NOTES" in the body](https://github.com/orgs/community/discussions/59734) - nicely done. 

🐙 **Issues and Projects Info**: All updates, info, releases, etc that are Issues and Projects-related!
* **The Latest Issues & Projects Update**: Copy and paste improvements, updates to the API in projects, and a whole lot of bug fixes. [Get all the info on this update from our changelog](https://github.blog/changelog/2023-06-22-github-issues-projects-june-22nd-update/). 
* **More Issues & Projects Updates**: Don’t miss this one - includes the hotly requested export to CSV capability, column limits and more. Check out the [changelog](https://github.blog/changelog/2023-06-15-github-issues-projects-june-15th-update/) for all that was included in this release. 
* **Quick Access to Projects on GitHub Mobile** 📱: You can now view all projects you're collaborating on directly from the home screen on GitHub Mobile. [View the changelog](https://github.blog/changelog/2023-06-27-quick-access-to-projects-on-github-mobile/) and leave us [feedback](https://github.com/orgs/community/discussions/59625) on the Community. 

⬆️ **Level up**: Resources to take your skills to new heights ☁️
* **Make your GitHub projects more accessible with accessibility-alt-text-bot**: The accessibility-alt-text-bot leaves automated reminders in a comment when a user shares an image without providing meaningful alt text. [Learn more in our blog](https://github.blog/2023-06-12-make-your-github-projects-more-accessible-with-accessibility-alt-text-bot/) about this development and how you can make your Issues more accessible 👐. 
* **Highlight Specific Code to Make an Issue**: From our Five things you didn’t know you could do with GitHub video, we show you how to share code with your team, [watch it here](https://www.youtube.com/watch?v=DWha-zTQpPE&t=995s). 

There are even more opportunities to share feedback this month and we’re all ears to hear yours!

**Comment by mtcA6** [comment]
THIS is going to level up what i can do while you all ship features. It's really a huge help to be able to export the data and create pivot tables if I need to nest a view in a deeper way than I can get w/the current UI

**Comment by queenofcorgis** [comment]
Our category name got a slight change, read about it my [check-in post](https://github.com/orgs/community/discussions/63398)!
